modules/ask_va_api/app/lib/ask_va_api/contents/entity.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+module AskVAApi
+  module Contents
+    class Entity
+      attr_reader :id,
+                  :name,
+                  :parent_id,
+                  :description,
+                  :requires_authentication,
+                  :allow_attachments,
+                  :rank_order,
+                  :display_name,
+                  :topic_type,
+                  :contact_preferences
+
+      def initialize(info)
```

Comment:
Are you ok with nil values here if any of these info attributes don't exist?

Reply:
Yes, it's not a hard requirement.